scripts/xen/gold_linux/README

```diff
@@ -15,7 +15,9 @@ echo "http://dl-3.alpinelinux.org/alpine/v3.9/main" >> /etc/apk/repositories
 # add stable community repository (required for docker installation)
 echo "http://dl-cdn.alpinelinux.org/alpine/latest-stable/community" >> /etc/apk/repositories
```

Comment:
At the time of writing https://mirrors.alpinelinux.org/ propoes `dl-cdn` as first choice, I think we'll stop at this.
I've change `dl-3` above to this.

But `latest-stable` looks like moved further. I get strange errors while trying add docker until I replaced this with `v3.9` as above.  

Just wanted to eliminate any surprises https://github.com/embox/embox/commit/d915e6a6f1235874858d45558eab1414642355a9